airflow/io/path.py

```diff
@@ -54,7 +54,7 @@ def __getattr__(self, name):
         if callable(attr):
             # If the attribute is a method, wrap it in another method to intercept the call
             def wrapper(*args, **kwargs):
-                self.log.error("Calling method: %s", name)
+                self.log.debug("Calling method: %s", name)
```

Comment:
Is this related to this PR?
If not please extract to separated PR

Reply:
@eladkal done. Mistakenly attached change from https://github.com/apache/airflow/pull/41615